paimon-format/src/main/java/org/apache/orc/OrcConf.java

```diff
@@ -0,0 +1,519 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.orc;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.hadoop.conf.Configuration;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Properties;
+
+/** Define the configuration properties that Orc understands. */
+public enum OrcConf {
```

Comment:
I encountered the class conflict in my local test, there may be problems with class with the same name.

```
Caused by: java.lang.NoSuchFieldError: COMPRESSION_ZSTD_LEVEL
	at org.apache.paimon.format.OrcOptions.<clinit>(OrcOptions.java:42)
	at org.apache.paimon.format.orc.OrcWriterFactory.create(OrcWriterFactory.java:132)
	at org.apache.paimon.io.SingleFileWriter.<init>(SingleFileWriter.java:68)
	at org.apache.paimon.io.StatsCollectingSingleFileWriter.<init>(StatsCollectingSingleFileWriter.java:57)
	at org.apache.paimon.io.RowDataFileWriter.<init>(RowDataFileWriter.java:57)
	at org.apache.paimon.io.RowDataRollingFileWriter.lambda$new$0(RowDataRollingFileWriter.java:53)
	at org.apache.paimon.io.RollingFileWriter.openCurrentWriter(RollingFileWriter.java:99)
	at org.apache.paimon.io.RollingFileWriter.write(RollingFileWriter.java:78)
	at org.apache.paimon.append.AppendOnlyWriter$DirectSinkWriter.write(AppendOnlyWriter.java:347)
	at org.apache.paimon.append.AppendOnlyWriter.write(AppendOnlyWriter.java:144)
	at org.apache.paimon.append.AppendOnlyWriter.write(AppendOnlyWriter.java:60)
	at org.apache.paimon.operation.AbstractFileStoreWrite.write(AbstractFileStoreWrite.java:126)
	at org.apache.paimon.table.sink.TableWriteImpl.writeAndReturn(TableWriteImpl.java:145)
	at org.apache.paimon.table.sink.TableWriteImpl.write(TableWriteImpl.java:126)
```

Reply:
https://github.com/apache/paimon/commit/0be4a21161fd1c6bc957907c9418e2b56417a111 fixed this.